modules/puppet/files/puppetize.sh

```diff
@@ -102,6 +104,32 @@ function set_osx_names()
     scutil --set LocalHostName $HOST || exit 1
 }
 
+# make sure the time is set correctly, or SSL will fail, badly.
+function sync_clock()
+{
+    case "$OS" in
+        Darwin)
+            if launchctl list org.ntp.ntpd > /dev/null 2>&1 ; then
+                launchctl unload /System/Library/LaunchDaemons/org.ntp.ntpd.plist
+            fi
+            ntpdate $NTP_SERVER
+            launchctl load -w /System/Library/LaunchDaemons/org.ntp.ntpd.plist
+            ;;
+
+        CentOS)
+            ntprunning=`ps ax | grep ntpd | grep -v grep`
```

Comment:
What if we always issue the service stop request and let that fail if it isn't running? Grepping over ps feels like we could make a mistake.

Or, do we have some servers not running ntpd? Maybe we can check service instead of looking for a string in the process list?

Reply:
I agree but it is slightly out of scope and I'm hesitant to change what is already working.

Reply:
+1 and I don't expect anything else to have the "ntpd" string in ps